### PR TITLE
Replace the deprecated gtk_widget_set_state with gtk_widget_set_state…

### PR DESCRIPTION
**Pull request by zhuyaliang**
Replace the deprecated gtk_widget_set_state with gtk_widget_set_state_flags

### CODE REVIEW

**Review by raveit65 — APPROVED** [review]
LGTM

File: libslab/app-shell.c [hunk 1]
```diff
@@ -464,7 +464,7 @@ populate_groups_section (AppShellData * app_data)
 		CategoryData *data = (CategoryData *) cat_list->data;
 		if (NULL != data->filtered_launcher_list)
 		{
-			gtk_widget_set_state (GTK_WIDGET (data->group_launcher), GTK_STATE_NORMAL);
+			gtk_widget_set_state_flags (GTK_WIDGET (data->group_launcher), GTK_STATE_NORMAL, FALSE);
```
Comment (vkareh) [inline libslab/app-shell.c:467]: `GTK_STATE_NORMAL` should be `GTK_STATE_FLAG_NORMAL`

File: libslab/app-shell.c [hunk 2]
```diff
@@ -623,7 +623,7 @@ generate_filtered_lists (gpointer catdata, gpointer user_data)
 
 		/* Since the filter may remove this entry from the
 		   container it will not get a mouse out event */
-		gtk_widget_set_state (GTK_WIDGET (launcher), GTK_STATE_NORMAL);
+		gtk_widget_set_state_flags (GTK_WIDGET (launcher), GTK_STATE_NORMAL, FALSE);
```
Comment (vkareh) [inline libslab/app-shell.c:626]: `GTK_STATE_NORMAL` should be `GTK_STATE_FLAG_NORMAL`

**Review by vkareh — CHANGES_REQUESTED** [review]

**Review by vkareh — APPROVED** [review]
LGTM

**Pull request merged**